data/operators/amenity/fast_food.json

```diff
@@ -0,0 +1,25 @@
+{
+  "properties": {
+    "path": "operators/amenity/fast_food",
+    "preserveTags": [],
+    "exclude": {
+      "generic": [
+        "^[Ff]ast food?$",
```

Comment:
AFAIK this will match:
Fast food
fast food
Fast foo
fast foo

Did you mean foods?